src/components/molecules/Loading/Loading.tsx

```diff
@@ -9,18 +9,19 @@ import "./Loading.scss";
 
 export interface LoadingProps extends ContainerClassName {
   label?: string;
+  displayWhile?: boolean;
```

Comment:
`displayWhile` display while what? The name is confusing. This can be simplified to `show` `visible` `isLoading`

Reply:
Idea is to have `displayWhile={isLoading}` in places where `{ isLoading && <Loading> }` might be an overkill of braces. Not meant to be <Loading show /> since it's `true` by default.